### ISSUE TITLE
Generating consistent features

### ISSUE THREAD

**Issue by piyushK52** [fudan-generative-vision/hallo#60]
I have noticed that teeth in the generated videos are always a hit or a miss, is it because of the nature of base ad? (that it finds it difficult to maintain consistency in smaller elements). Also, do you guys know how this can be avoided/minimized ?

https://github.com/fudan-generative-vision/hallo/assets/34690994/b0c4291a-452b-4e7c-bae5-4c8a12ae2d4c

**Comment by subazinga** [comment]
Yes, that a known problem. We are still fixing it. Currently, maybe [gfpgan](https://github.com/TencentARC/GFPGAN) can help. It is also used by SadTalker.

**Comment by piyushK52** [comment]
Nice! Looking forward to it
